### PR TITLE
1377: Drag & Drop Not Intuitive

### PR DESCRIPTION
**Pull request by lilitkarapetyan**
## 🧰 Issue

#1377

## 🔨Work carried out:

- [x] Tests pass

## Confirmations

- [ ] I have chosen reviewers for my PR.
- [ ] I have assigned myself to this PR.
- [ ] I have chosen an appropriate label for the PR.
- [ ] I have completed the mandatory sections of this document.
- [ ] I have deleted any unused sections.
- [ ] I confirm that I have checked for required README updates and acted accordingly.

### CODE REVIEW

**Comment by IanMayo** [comment]
Thanks @lilitkarapetyan - but I think we need a confirm dialog to delete the asset after clicking the red x, please.

**Comment by ShendMikullovci** [comment]
Thanks @lilitkarapetyan! Could you make the X dark grey and center it on the asset? It'd look something like this: 

Asset Delete Feature

As you can see, the selected asset has a slightly lighter grey to bring it into focus since the asset selection background is navy blue.

File: packages/client/src/Components/Modals/DeleteConfirmation.jsx [hunk 1]
```diff
@@ -23,26 +23,32 @@ const DeleteModal = () => {
     const { type, data } = currentModal.data
     const curTab = wargame.currentTab
```
Comment (IanMayo) [inline packages/client/src/Components/Modals/DeleteConfirmation.jsx:25]: @lilitkarapetyan -  I've switched to a select, so we have a `default` branch.

Do you have any objection to this?

**Comment by ShendMikullovci** [comment]
Just checked out the deployment. Looks much better!

**Review by IanMayo — APPROVED** [review]

**Comment by IanMayo** [comment]
@lilitkarapetyan - I think that the change to the formatting for `List of assets` in this PR broke the display of `asset-icons` in the top-left panel:
image

**Pull request merged**